src/Security/RolesBuilder/AdminRolesBuilder.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\UserBundle\Security\RolesBuilder;
+
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Sonata\AdminBundle\Admin\Pool;
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * @author Silas Joisten <silasjoisten@hotmail.de>
+ */
+final class AdminRolesBuilder implements AdminRolesBuilderInterface
+{
+    /**
+     * @var AuthorizationCheckerInterface
+     */
+    private $authorizationChecker;
+
+    /**
+     * @var Pool
+     */
+    private $pool;
+
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * @var string []
+     */
+    private $excludeAdmin = [];
```

Comment:
Name is still singular but it is an array

Reply:
@silasjoisten please fix the naming to plural